### PR TITLE
Greatly improves the explosions from the alkali perspiration symptom (and also a plushie shitpost bit because I accidentally updated my master branch twice)

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR reorders alkali perspiration symptom's effects, modifies the explosion details of the explosions from it (and allows them to occur in both stage 4 and stage 5), and modifies its threshold effect descriptions (to better reflect what they do). Oh, and it fixes an obscure bug that would cause Alkali_fire_stage_5() to trigger twice in one cycle if the water reaction explosion (from the alkali perspiration symptom) proc'd that cycle.

EDIT: I, uh, forgot to make a new branch for both this PR and my plushie PR, so, uh, I guess they're both one PR now. Trying to kill a "divine" plushie will now cause a Dagoth Ur/Morrowind reference to occur.

## Why It's Good For The Game

The alkali perspiration symptom has the unique effect of causing its virus's hosts to explode if they're wet (as in, they have a negative number of fire stacks) when the symptom activates. Unfortunately, this effect is rarely seen, as the alkali perspiration symptom applies its positive fire stacks (and tries to set its virus's host on fire) BEFORE the check to see if the host has a negative number of fire stacks takes place. This means that at stage 5, you have to have less than -8 fire stacks (less than *-16* fire stacks if the resistance threshold is met) in order for the alkali perspiration symptom to cause you to explode (and this is assuming that the virus hasn't met the transmission threshold that causes it to synthesize napalm and clf3 in hosts (I don't actually know if napalm is able to apply its reaction_mob effect before the negative fire stack check runs, but it doesn't really matter all that much)). For reference, the fire stack caps are -20 and +20, and living mobs naturally gain one fire stack per tick of Life() while they have a negative fire stack total. The alkali perspiration symptom checks for negative fire stacks once every 30-90 seconds (even LESS often if you meet the resistance threshold), which means that you could actually hit the negative fire stack cap and naturally return to a fire stack total of 0 without the symptom detecting your wetness (this PR doesn't actually fix that, but I thought that I should mention it). In other words, unless you're standing directly under a shower and either wait a while or are unlucky, the alkali perspiration symptom will never cause you to explode.

This is actually a good thing, since (as far as I'm aware) the explosions from the alkali perspiration symptom currently GIB THE HOSTS THAT THEY ORIGINATE FROM (they apply a devastating explosion effect to the host's tile), which seems incredibly unfun to be on the receiving end of (and before you say "i ded", I've never actually seen this happen before (due to the aforementioned difficulty to even encounter this effect unless you're standing under a shower or something)).

Also, for some strange reason, when the water reaction explosion DOES get triggered, Alkali_fire_stage_5() triggers a second time (in one cycle).

This PR makes the alkali perspiration symptom check for negative fire stacks BEFORE giving you fire stacks (and chems) and nerfs the actual explosion itself to be just a light explosion (with a heavy explosion effect only occurring on the host's tile if the stage speed 8 threshold is met). It also fixes the double Alkali_fire_stage_5() triggering bug and modifies the descriptions of the alkali perspiration symptom's threshold effects. It also lets the water reaction explosion occur in both stage 4 AND stage 5, instead of just in stage 5. Yes, this paragraph is basically a reiteration of the About The Pull Request (shouldn't that be "About the Pull Request"?) paragraph with some more details, but the details in this paragraph made less sense without the explanations in this Why It's Good For The Game section.

EDIT: Dagothposting is also fun and cool.

## Changelog
:cl: ATHATH
tweak: Made the alkali perspiration symptom's water reaction explosion effect much easier to proc.
balance: The alkali perspiration symptom's water reaction explosion effect no longer gibs its virus's host. Instead, it applies a light explosion effect to its virus's hosts tile and to tiles near them. The stage speed 8 threshold effect now also increases the potency of the explosion effect to a "heavy" level on the host's tile.
balance: The alkali perspiration symptom's water reaction explosion effect can now occur in both stage 4 and in stage 5 (instead of only in stage 5).
fix: Fixes a bug that would cause the alkali perspiration symptom's water reaction explosion effect to trigger the alkali perspiration symptom's Alkali_fire_stage_5() effect a second time in a single cycle.
tweak: Slightly modifies the alkali perspiration symptom's threshold effect descriptions to be slightly more clear about what its threshold effects actually do.
add: Adds a Morrowind reference that occurs if you try to kill a divine plushie.
/:cl:

### CODE REVIEW

**Comment by Adanps** [comment]
The explosions will still probably be rare because it seems people will usually dry off before they catch on fire. Still, maybe now it will actually be relevant in situations other than infected players standing under a shower head.

**Comment by ATH1909** [comment]
Aw fuck I forgot to make a new branch and accidentally updated master.

I guess this is now a dual-purpose PR.

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -118,9 +119,22 @@
...
-			user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
-			I.play_tool_sound(src)
-			stuffed = FALSE
+			if(!divine)
+				user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
+				I.play_tool_sound(src)
+				stuffed = FALSE
+			else
+				to_chat(user, "<span class='notice'>What a fool you are. [src] is a god, how can you kill a god? What a grand and intoxicating innocence.</span>")
+				if(iscarbon(user) && user.drunkenness < 50)
```
Comment (ATH1909) [inline code/game/objects/items/plushes.dm]: it seems like the automatic checks don't like me referencing the drunkenness variable with a living mob, even though I checked for carbon-ness beforehand

is there a way around this? like, a way to temporarily typecast user as a carbon or something?

**Comment by ATH1909** [comment]
Wait, is the plural of "plushie" "plushies" or "plushes"?

**Comment by ATH1909** [comment]
Oh, and before you ask, I'm using the in-browser version of github, so I I don't think that I can just input a command to undo my alterations to my master branch without doing weird stuff to some of my existing PRs (maybe I can, though? I dunno).

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -22,6 +22,7 @@
 	var/heartbroken = FALSE
 	var/vowbroken = FALSE
 	var/young = FALSE
+	var/divine = FALSE
```
Comment (wesoda25) [inline code/game/objects/items/plushes.dm]: if you wanna stay on web editor you could just make a separate commit reversing all this plushie stuff

Comment (ATH1909) [reply code/game/objects/items/plushes.dm]: Alright, I'll do that tomorrow morning.

**Comment by ExcessiveUseOfCobblestone** [comment]
bro get off the webbitor

**Comment by ATH1909** [comment]
I've now separated this PR into two separate PRs:
https://github.com/tgstation/tgstation/pull/46062
https://github.com/tgstation/tgstation/pull/46063

I'm going to close this PR now, since those two PRs do what this PR does.
